templates/index.html

```diff
@@ -0,0 +1,13 @@
+<html>
+<head>
```

Comment:
I believe standard style is to indent one level from <HTML> but I could be wrong